src/lib/viewers/BaseViewer.js

```diff
@@ -1,5 +1,4 @@
 import 'box-annotations/lib/Annotator.scss';
```

Comment:
I think once we bundle annotations with webpack properly we should be able to get rid of this CSS import in BaseViewer.js

Reply:
Will change this once I can test with yours.